nohup: ignoring input
[2025-06-10 19:24:06 -0500] [679494] [INFO] Starting gunicorn 23.0.0
[2025-06-10 19:24:06 -0500] [679494] [INFO] Listening at: http://0.0.0.0:8000 (679494)
[2025-06-10 19:24:06 -0500] [679494] [INFO] Using worker: sync
[2025-06-10 19:24:06 -0500] [679495] [INFO] Booting worker with pid: 679495
[2025-06-10 19:24:06 -0500] [679496] [INFO] Booting worker with pid: 679496
[2025-06-10 19:24:06 -0500] [679497] [INFO] Booting worker with pid: 679497
[2025-06-10 19:24:14 -0500] [679494] [INFO] Handling signal: hup
[2025-06-10 19:24:14 -0500] [679494] [INFO] Hang up: Master
[2025-06-10 19:24:14 -0500] [679500] [INFO] Booting worker with pid: 679500
[2025-06-10 19:24:14 -0500] [679501] [INFO] Booting worker with pid: 679501
[2025-06-10 19:24:14 -0500] [679495] [INFO] Worker exiting (pid: 679495)
[2025-06-10 19:24:14 -0500] [679496] [INFO] Worker exiting (pid: 679496)
[2025-06-10 19:24:14 -0500] [679497] [INFO] Worker exiting (pid: 679497)
[2025-06-10 19:24:14 -0500] [679502] [INFO] Booting worker with pid: 679502
[2025-06-10 19:24:14 -0500] [679494] [ERROR] Worker (pid:679495) was sent SIGTERM!
[2025-06-10 19:24:14 -0500] [679494] [ERROR] Worker (pid:679496) was sent SIGTERM!
[2025-06-10 19:24:14 -0500] [679494] [ERROR] Worker (pid:679497) was sent SIGTERM!
[2025-06-10 19:25:25 -0500] [679494] [INFO] Handling signal: hup
[2025-06-10 19:25:25 -0500] [679494] [INFO] Hang up: Master
[2025-06-10 19:25:25 -0500] [679515] [INFO] Booting worker with pid: 679515
[2025-06-10 19:25:25 -0500] [679516] [INFO] Booting worker with pid: 679516
[2025-06-10 19:25:25 -0500] [679500] [INFO] Worker exiting (pid: 679500)
[2025-06-10 19:25:25 -0500] [679501] [INFO] Worker exiting (pid: 679501)
[2025-06-10 19:25:25 -0500] [679502] [INFO] Worker exiting (pid: 679502)
[2025-06-10 19:25:25 -0500] [679517] [INFO] Booting worker with pid: 679517
[2025-06-10 19:25:26 -0500] [679494] [ERROR] Worker (pid:679501) was sent SIGTERM!
[2025-06-10 19:25:26 -0500] [679494] [ERROR] Worker (pid:679500) was sent SIGTERM!
[2025-06-10 19:25:26 -0500] [679494] [ERROR] Worker (pid:679502) was sent SIGTERM!
[2025-06-10 19:26:05 -0500] [679494] [INFO] Handling signal: hup
[2025-06-10 19:26:05 -0500] [679494] [INFO] Hang up: Master
[2025-06-10 19:26:05 -0500] [679879] [INFO] Booting worker with pid: 679879
[2025-06-10 19:26:05 -0500] [679880] [INFO] Booting worker with pid: 679880
[2025-06-10 19:26:05 -0500] [679515] [INFO] Worker exiting (pid: 679515)
[2025-06-10 19:26:06 -0500] [679516] [INFO] Worker exiting (pid: 679516)
[2025-06-10 19:26:06 -0500] [679517] [INFO] Worker exiting (pid: 679517)
[2025-06-10 19:26:06 -0500] [679881] [INFO] Booting worker with pid: 679881
[2025-06-10 19:26:06 -0500] [679494] [ERROR] Worker (pid:679515) was sent SIGTERM!
[2025-06-10 19:26:06 -0500] [679494] [ERROR] Worker (pid:679516) was sent SIGTERM!
[2025-06-10 19:26:06 -0500] [679494] [ERROR] Worker (pid:679517) was sent SIGTERM!
[2025-06-10 19:27:12 -0500] [679494] [INFO] Handling signal: hup
[2025-06-10 19:27:12 -0500] [679494] [INFO] Hang up: Master
[2025-06-10 19:27:12 -0500] [679886] [INFO] Booting worker with pid: 679886
[2025-06-10 19:27:12 -0500] [679887] [INFO] Booting worker with pid: 679887
[2025-06-10 19:27:12 -0500] [679881] [INFO] Worker exiting (pid: 679881)
[2025-06-10 19:27:12 -0500] [679879] [INFO] Worker exiting (pid: 679879)
[2025-06-10 19:27:12 -0500] [679880] [INFO] Worker exiting (pid: 679880)
[2025-06-10 19:27:12 -0500] [679888] [INFO] Booting worker with pid: 679888
[2025-06-10 19:27:14 -0500] [679494] [ERROR] Worker (pid:679881) was sent SIGTERM!
[2025-06-10 19:27:14 -0500] [679494] [ERROR] Worker (pid:679879) was sent SIGTERM!
[2025-06-10 19:27:14 -0500] [679494] [ERROR] Worker (pid:679880) was sent SIGTERM!
[2025-06-10 19:34:49 -0500] [679494] [INFO] Handling signal: hup
[2025-06-10 19:34:49 -0500] [679494] [INFO] Hang up: Master
[2025-06-10 19:34:49 -0500] [679957] [INFO] Booting worker with pid: 679957
[2025-06-10 19:34:49 -0500] [679958] [INFO] Booting worker with pid: 679958
[2025-06-10 19:34:49 -0500] [679887] [INFO] Worker exiting (pid: 679887)
[2025-06-10 19:34:49 -0500] [679886] [INFO] Worker exiting (pid: 679886)
[2025-06-10 19:34:49 -0500] [679888] [INFO] Worker exiting (pid: 679888)
[2025-06-10 19:34:49 -0500] [679959] [INFO] Booting worker with pid: 679959
[2025-06-10 19:34:50 -0500] [679494] [ERROR] Worker (pid:679888) was sent SIGTERM!
--- Logging error ---
[2025-06-10 19:34:50 -0500] [679494] [ERROR] Worker (pid:679886) was sent SIGTERM!
Traceback (most recent call last):
  File "/usr/lib/python3.12/logging/__init__.py", line 1163, in emit
    stream.write(msg + self.terminator)
RuntimeError: reentrant call inside <_io.BufferedWriter name='<stderr>'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.12/logging/__init__.py", line 1163, in emit
    stream.write(msg + self.terminator)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 241, in handle_chld
    self.reap_workers()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 553, in reap_workers
    self.log.error(msg)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/glogging.py", line 271, in error
    self.error_log.error(msg, *args, **kwargs)
  File "/usr/lib/python3.12/logging/__init__.py", line 1568, in error
    self._log(ERROR, msg, args, **kwargs)
  File "/usr/lib/python3.12/logging/__init__.py", line 1684, in _log
    self.handle(record)
  File "/usr/lib/python3.12/logging/__init__.py", line 1700, in handle
    self.callHandlers(record)
  File "/usr/lib/python3.12/logging/__init__.py", line 1762, in callHandlers
    hdlr.handle(record)
  File "/usr/lib/python3.12/logging/__init__.py", line 1028, in handle
    self.emit(record)
  File "/usr/lib/python3.12/logging/__init__.py", line 1168, in emit
    self.handleError(record)
  File "/usr/lib/python3.12/logging/__init__.py", line 1081, in handleError
    sys.stderr.write('--- Logging error ---\n')
RuntimeError: reentrant call inside <_io.BufferedWriter name='<stderr>'>
Call stack:
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/bin/gunicorn", line 8, in <module>
    sys.exit(run())
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/app/wsgiapp.py", line 66, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]", prog=prog).run()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/app/base.py", line 235, in run
    super().run()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/app/base.py", line 71, in run
    Arbiter(self).run()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 208, in run
    self.sleep()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 359, in sleep
    ready = select.select([self.PIPE[0]], [], [], 1.0)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 241, in handle_chld
    self.reap_workers()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 553, in reap_workers
    self.log.error(msg)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/glogging.py", line 271, in error
    self.error_log.error(msg, *args, **kwargs)
Message: 'Worker (pid:679888) was sent SIGTERM!'
Arguments: ()
[2025-06-10 20:25:17 -0500] [679959] [WARNING] Invalid request from ip=193.188.192.219: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-10 21:13:15 -0500] [679958] [WARNING] Invalid request from ip=206.168.34.118: Invalid HTTP Version: (2, 0)
[2025-06-10 21:14:18 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:679959)
[2025-06-10 21:14:18 -0500] [679959] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-10 21:14:18 -0500] [679959] [INFO] Worker exiting (pid: 679959)
[2025-06-10 21:14:18 -0500] [684362] [INFO] Booting worker with pid: 684362
[2025-06-11 09:27:01 -0500] [679957] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-11 11:38:15 -0500] [684362] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-11 22:45:00 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:679958)
[2025-06-11 22:45:00 -0500] [679958] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 271, in parse
    self.get_data(unreader, buf, stop=True)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-11 22:45:00 -0500] [679958] [INFO] Worker exiting (pid: 679958)
[2025-06-11 22:45:00 -0500] [749275] [INFO] Booting worker with pid: 749275
[2025-06-12 03:01:18 -0500] [749275] [WARNING] Invalid request from ip=206.168.34.209: Invalid HTTP Version: (2, 0)
[2025-06-12 03:02:24 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:749275)
[2025-06-12 03:02:24 -0500] [749275] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-12 03:02:24 -0500] [749275] [INFO] Worker exiting (pid: 749275)
[2025-06-12 03:02:24 -0500] [759796] [INFO] Booting worker with pid: 759796
[2025-06-12 08:26:16 -0500] [679957] [WARNING] Invalid request from ip=165.154.182.182: Invalid HTTP request line: ''
[2025-06-12 09:39:22 -0500] [679957] [WARNING] Invalid request from ip=3.132.23.201: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-12 13:52:26 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:684362)
[2025-06-12 13:52:26 -0500] [684362] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 271, in parse
    self.get_data(unreader, buf, stop=True)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-12 13:52:26 -0500] [684362] [INFO] Worker exiting (pid: 684362)
[2025-06-12 13:52:27 -0500] [787626] [INFO] Booting worker with pid: 787626
[2025-06-12 14:17:19 -0500] [679957] [WARNING] Invalid request from ip=194.180.49.39: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-13 02:21:18 -0500] [787626] [WARNING] Invalid request from ip=206.168.34.200: Invalid HTTP Version: (2, 0)
[2025-06-13 02:22:13 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:759796)
[2025-06-13 02:22:13 -0500] [759796] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-13 02:22:13 -0500] [759796] [INFO] Worker exiting (pid: 759796)
[2025-06-13 02:22:14 -0500] [818756] [INFO] Booting worker with pid: 818756
[2025-06-13 11:03:03 -0500] [787626] [WARNING] Invalid request from ip=167.94.145.108: Invalid HTTP Version: (2, 0)
[2025-06-14 02:46:13 -0500] [679957] [WARNING] Invalid request from ip=167.94.138.174: Invalid HTTP Version: (2, 0)
[2025-06-14 02:47:23 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:818756)
[2025-06-14 02:47:23 -0500] [818756] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-14 02:47:23 -0500] [818756] [INFO] Worker exiting (pid: 818756)
[2025-06-14 02:47:24 -0500] [879979] [INFO] Booting worker with pid: 879979
[2025-06-14 10:28:48 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:787626)
[2025-06-14 10:28:48 -0500] [787626] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-14 10:28:48 -0500] [787626] [INFO] Worker exiting (pid: 787626)
[2025-06-14 10:28:48 -0500] [899952] [INFO] Booting worker with pid: 899952
[2025-06-14 10:29:21 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:899952)
[2025-06-14 10:29:21 -0500] [899952] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-14 10:29:21 -0500] [899952] [INFO] Worker exiting (pid: 899952)
[2025-06-14 10:29:21 -0500] [899953] [INFO] Booting worker with pid: 899953
[2025-06-14 16:08:52 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:879979)
[2025-06-14 16:08:52 -0500] [879979] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-14 16:08:52 -0500] [879979] [INFO] Worker exiting (pid: 879979)
[2025-06-14 16:08:52 -0500] [913970] [INFO] Booting worker with pid: 913970
[2025-06-14 21:32:43 -0500] [679957] [WARNING] Invalid request from ip=162.142.125.219: Invalid HTTP Version: (2, 0)
[2025-06-14 21:33:25 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:679957)
[2025-06-14 21:33:25 -0500] [679957] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-14 21:33:25 -0500] [679957] [INFO] Worker exiting (pid: 679957)
[2025-06-14 21:33:25 -0500] [926881] [INFO] Booting worker with pid: 926881
[2025-06-15 13:10:02 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:899953)
[2025-06-15 13:10:02 -0500] [899953] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-15 13:10:02 -0500] [899953] [INFO] Worker exiting (pid: 899953)
[2025-06-15 13:10:02 -0500] [965631] [INFO] Booting worker with pid: 965631
[2025-06-15 14:35:03 -0500] [926881] [WARNING] Invalid request from ip=18.220.154.78: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-15 18:20:29 -0500] [913970] [WARNING] Invalid request from ip=167.94.145.106: Invalid HTTP Version: (2, 0)
[2025-06-16 12:29:18 -0500] [926881] [WARNING] Invalid request from ip=3.137.73.221: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-16 15:48:16 -0500] [926881] [WARNING] Invalid request from ip=206.168.34.201: Invalid HTTP Version: (2, 0)
[2025-06-16 15:49:15 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:965631)
[2025-06-16 15:49:15 -0500] [965631] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-16 15:49:15 -0500] [965631] [INFO] Worker exiting (pid: 965631)
[2025-06-16 15:49:15 -0500] [1030692] [INFO] Booting worker with pid: 1030692
[2025-06-17 01:11:57 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:926881)
[2025-06-17 01:11:57 -0500] [926881] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-17 01:11:57 -0500] [926881] [INFO] Worker exiting (pid: 926881)
[2025-06-17 01:11:58 -0500] [1053020] [INFO] Booting worker with pid: 1053020
[2025-06-17 02:38:08 -0500] [913970] [WARNING] Invalid request from ip=199.45.155.76: Invalid HTTP Version: (2, 0)
[2025-06-17 04:09:00 -0500] [1053020] [WARNING] Invalid request from ip=167.94.145.108: Invalid HTTP Version: (2, 0)
[2025-06-17 15:10:42 -0500] [913970] [WARNING] Invalid request from ip=176.125.228.15: Invalid HTTP request line: 'SSH-2.0-WanScannerBot'
[2025-06-17 15:10:43 -0500] [913970] [WARNING] Invalid request from ip=176.125.228.15: Invalid HTTP request line: '\x03\x00\x00&!Ã \x00\x00Ã¾Ã\x00Cookie: mstshash='
[2025-06-17 15:10:57 -0500] [913970] [WARNING] Invalid request from ip=176.125.228.15: Invalid HTTP Version: 'RTSP/1.0'
[2025-06-17 15:11:02 -0500] [1053020] [WARNING] Invalid request from ip=176.125.228.15: Invalid HTTP request line: '\x01\x00\x00\x00'
[2025-06-17 19:23:56 -0500] [1030692] [WARNING] Invalid request from ip=165.154.41.152: Invalid HTTP request line: ''
[2025-06-17 22:57:42 -0500] [1053020] [WARNING] Invalid request from ip=3.137.73.221: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-18 10:28:06 -0500] [913970] [WARNING] Invalid request from ip=162.142.125.117: Invalid HTTP Version: (2, 0)
[2025-06-18 10:29:02 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:913970)
[2025-06-18 10:29:02 -0500] [913970] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-18 10:29:02 -0500] [913970] [INFO] Worker exiting (pid: 913970)
[2025-06-18 10:29:03 -0500] [1136760] [INFO] Booting worker with pid: 1136760
[2025-06-18 10:31:41 -0500] [1136760] [WARNING] Invalid request from ip=206.168.34.195: Invalid HTTP Version: (2, 0)
[2025-06-18 10:33:06 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1136760)
[2025-06-18 10:33:06 -0500] [1136760] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-18 10:33:06 -0500] [1136760] [INFO] Worker exiting (pid: 1136760)
[2025-06-18 10:33:06 -0500] [1136784] [INFO] Booting worker with pid: 1136784
[2025-06-18 16:25:48 -0500] [1136784] [WARNING] Invalid request from ip=91.199.163.12: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-18 18:32:17 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1030692)
[2025-06-18 18:32:17 -0500] [1030692] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-18 18:32:17 -0500] [1030692] [INFO] Worker exiting (pid: 1030692)
[2025-06-18 18:32:17 -0500] [1157024] [INFO] Booting worker with pid: 1157024
[2025-06-18 19:32:47 -0500] [1136784] [WARNING] Invalid request from ip=91.199.163.12: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-18 19:55:10 -0500] [1157024] [WARNING] Invalid request from ip=45.140.17.107: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-18 23:39:23 -0500] [1157024] [WARNING] Invalid request from ip=91.199.163.12: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-19 01:22:36 -0500] [1136784] [WARNING] Invalid request from ip=45.140.17.107: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-19 02:06:30 -0500] [1157024] [WARNING] Invalid request from ip=3.138.185.30: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-19 12:06:46 -0500] [1053020] [WARNING] Invalid request from ip=199.45.154.114: Invalid HTTP Version: (2, 0)
[2025-06-20 12:27:40 -0500] [1053020] [WARNING] Invalid request from ip=167.94.145.111: Invalid HTTP Version: (2, 0)
[2025-06-20 12:53:37 -0500] [1157024] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-20 12:54:33 -0500] [1053020] [WARNING] Invalid request from ip=167.94.138.121: Invalid HTTP Version: (2, 0)
[2025-06-20 12:55:47 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1157024)
[2025-06-20 12:55:47 -0500] [1157024] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-20 12:55:47 -0500] [1157024] [INFO] Worker exiting (pid: 1157024)
[2025-06-20 12:55:48 -0500] [1262618] [INFO] Booting worker with pid: 1262618
[2025-06-20 13:52:22 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1136784)
[2025-06-20 13:52:22 -0500] [1136784] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-20 13:52:22 -0500] [1136784] [INFO] Worker exiting (pid: 1136784)
[2025-06-20 13:52:22 -0500] [1264651] [INFO] Booting worker with pid: 1264651
[2025-06-20 15:18:12 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:18:12 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:18:12 -0500] [1268588] [INFO] Booting worker with pid: 1268588
[2025-06-20 15:18:12 -0500] [1268589] [INFO] Booting worker with pid: 1268589
[2025-06-20 15:18:12 -0500] [1268590] [INFO] Booting worker with pid: 1268590
[2025-06-20 15:18:12 -0500] [1264651] [INFO] Worker exiting (pid: 1264651)
[2025-06-20 15:18:12 -0500] [1053020] [INFO] Worker exiting (pid: 1053020)
[2025-06-20 15:18:12 -0500] [1262618] [INFO] Worker exiting (pid: 1262618)
[2025-06-20 15:18:13 -0500] [679494] [ERROR] Worker (pid:1262618) was sent SIGTERM!
[2025-06-20 15:18:13 -0500] [679494] [ERROR] Worker (pid:1264651) was sent SIGTERM!
[2025-06-20 15:18:13 -0500] [679494] [ERROR] Worker (pid:1053020) was sent SIGTERM!
[2025-06-20 15:23:41 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:23:41 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:23:41 -0500] [1268632] [INFO] Booting worker with pid: 1268632
[2025-06-20 15:23:41 -0500] [1268589] [INFO] Worker exiting (pid: 1268589)
[2025-06-20 15:23:41 -0500] [1268588] [INFO] Worker exiting (pid: 1268588)
[2025-06-20 15:23:41 -0500] [1268633] [INFO] Booting worker with pid: 1268633
[2025-06-20 15:23:41 -0500] [1268590] [INFO] Worker exiting (pid: 1268590)
[2025-06-20 15:23:41 -0500] [1268634] [INFO] Booting worker with pid: 1268634
[2025-06-20 15:23:41 -0500] [679494] [ERROR] Worker (pid:1268589) was sent SIGTERM!
[2025-06-20 15:23:41 -0500] [679494] [ERROR] Worker (pid:1268588) was sent SIGTERM!
[2025-06-20 15:23:41 -0500] [679494] [ERROR] Worker (pid:1268590) was sent SIGTERM!
[2025-06-20 15:28:38 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:28:38 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:28:38 -0500] [1269082] [INFO] Booting worker with pid: 1269082
[2025-06-20 15:28:38 -0500] [1269083] [INFO] Booting worker with pid: 1269083
[2025-06-20 15:28:38 -0500] [1268632] [INFO] Worker exiting (pid: 1268632)
[2025-06-20 15:28:38 -0500] [1268633] [INFO] Worker exiting (pid: 1268633)
[2025-06-20 15:28:38 -0500] [1268634] [INFO] Worker exiting (pid: 1268634)
[2025-06-20 15:28:38 -0500] [1269084] [INFO] Booting worker with pid: 1269084
[2025-06-20 15:28:39 -0500] [679494] [ERROR] Worker (pid:1268633) was sent SIGTERM!
[2025-06-20 15:28:39 -0500] [679494] [ERROR] Worker (pid:1268634) was sent SIGTERM!
[2025-06-20 15:39:27 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:39:27 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:39:27 -0500] [1269501] [INFO] Booting worker with pid: 1269501
[2025-06-20 15:39:28 -0500] [1269502] [INFO] Booting worker with pid: 1269502
[2025-06-20 15:39:28 -0500] [1269082] [INFO] Worker exiting (pid: 1269082)
[2025-06-20 15:39:28 -0500] [1269083] [INFO] Worker exiting (pid: 1269083)
[2025-06-20 15:39:28 -0500] [1269084] [INFO] Worker exiting (pid: 1269084)
[2025-06-20 15:39:28 -0500] [1269503] [INFO] Booting worker with pid: 1269503
[2025-06-20 15:39:28 -0500] [679494] [ERROR] Worker (pid:1269082) was sent SIGTERM!
[2025-06-20 15:39:28 -0500] [679494] [ERROR] Worker (pid:1269083) was sent SIGTERM!
[2025-06-20 15:39:28 -0500] [679494] [ERROR] Worker (pid:1269084) was sent SIGTERM!
[2025-06-20 15:39:33,359] ERROR in app: Exception on /historiaIOT [GET]
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/app.py", line 138, in historia_iot
    return render_template('historia.html', registros=registros)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1295, in render
    self.environment.handle_exception()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/templates/historia.html", line 86, in top-level template code
    <td>{{ r.fecha.strftime('%Y-%m-%d %H:%M:%S') }}</td>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/jinja2/environment.py", line 490, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'r' is undefined
[2025-06-20 15:39:44,592] ERROR in app: Exception on /historiaIOT [GET]
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/app.py", line 138, in historia_iot
    return render_template('historia.html', registros=registros)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1295, in render
    self.environment.handle_exception()
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/templates/historia.html", line 86, in top-level template code
    <td>{{ r.fecha.strftime('%Y-%m-%d %H:%M:%S') }}</td>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/jinja2/environment.py", line 490, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'r' is undefined
[2025-06-20 15:42:12 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:42:12 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:42:12 -0500] [1269532] [INFO] Booting worker with pid: 1269532
[2025-06-20 15:42:12 -0500] [1269533] [INFO] Booting worker with pid: 1269533
[2025-06-20 15:42:12 -0500] [1269501] [INFO] Worker exiting (pid: 1269501)
[2025-06-20 15:42:12 -0500] [1269502] [INFO] Worker exiting (pid: 1269502)
[2025-06-20 15:42:12 -0500] [1269534] [INFO] Booting worker with pid: 1269534
[2025-06-20 15:42:12 -0500] [1269503] [INFO] Worker exiting (pid: 1269503)
[2025-06-20 15:42:13 -0500] [679494] [ERROR] Worker (pid:1269502) was sent SIGTERM!
[2025-06-20 15:42:13 -0500] [679494] [ERROR] Worker (pid:1269501) was sent SIGTERM!
[2025-06-20 15:42:13 -0500] [679494] [ERROR] Worker (pid:1269503) was sent SIGTERM!
[2025-06-20 15:46:23 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:46:23 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:46:23 -0500] [1269930] [INFO] Booting worker with pid: 1269930
[2025-06-20 15:46:23 -0500] [1269931] [INFO] Booting worker with pid: 1269931
[2025-06-20 15:46:23 -0500] [1269532] [INFO] Worker exiting (pid: 1269532)
[2025-06-20 15:46:23 -0500] [1269534] [INFO] Worker exiting (pid: 1269534)
[2025-06-20 15:46:23 -0500] [1269533] [INFO] Worker exiting (pid: 1269533)
[2025-06-20 15:46:23 -0500] [1269932] [INFO] Booting worker with pid: 1269932
[2025-06-20 15:52:20 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:52:20 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:52:20 -0500] [1269961] [INFO] Booting worker with pid: 1269961
[2025-06-20 15:52:20 -0500] [1269962] [INFO] Booting worker with pid: 1269962
[2025-06-20 15:52:20 -0500] [1269931] [INFO] Worker exiting (pid: 1269931)
[2025-06-20 15:52:20 -0500] [1269930] [INFO] Worker exiting (pid: 1269930)
[2025-06-20 15:52:20 -0500] [1269932] [INFO] Worker exiting (pid: 1269932)
[2025-06-20 15:52:20 -0500] [1269963] [INFO] Booting worker with pid: 1269963
[2025-06-20 15:58:06 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 15:58:06 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 15:58:06 -0500] [1270361] [INFO] Booting worker with pid: 1270361
[2025-06-20 15:58:06 -0500] [1269961] [INFO] Worker exiting (pid: 1269961)
[2025-06-20 15:58:06 -0500] [1270362] [INFO] Booting worker with pid: 1270362
[2025-06-20 15:58:06 -0500] [1269963] [INFO] Worker exiting (pid: 1269963)
[2025-06-20 15:58:06 -0500] [1269962] [INFO] Worker exiting (pid: 1269962)
[2025-06-20 15:58:06 -0500] [1270363] [INFO] Booting worker with pid: 1270363
[2025-06-20 16:00:53 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 16:00:53 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 16:00:53 -0500] [1270377] [INFO] Booting worker with pid: 1270377
[2025-06-20 16:00:53 -0500] [1270378] [INFO] Booting worker with pid: 1270378
[2025-06-20 16:00:53 -0500] [1270361] [INFO] Worker exiting (pid: 1270361)
[2025-06-20 16:00:53 -0500] [1270362] [INFO] Worker exiting (pid: 1270362)
[2025-06-20 16:00:53 -0500] [1270363] [INFO] Worker exiting (pid: 1270363)
[2025-06-20 16:00:53 -0500] [1270379] [INFO] Booting worker with pid: 1270379
[2025-06-20 16:00:53 -0500] [679494] [ERROR] Worker (pid:1270361) was sent SIGTERM!
[2025-06-20 16:00:53 -0500] [679494] [ERROR] Worker (pid:1270362) was sent SIGTERM!
[2025-06-20 16:00:53 -0500] [679494] [ERROR] Worker (pid:1270363) was sent SIGTERM!
[2025-06-20 17:48:18 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 17:48:18 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 17:48:18 -0500] [1275172] [INFO] Booting worker with pid: 1275172
[2025-06-20 17:48:18 -0500] [1275173] [INFO] Booting worker with pid: 1275173
[2025-06-20 17:48:18 -0500] [1270378] [INFO] Worker exiting (pid: 1270378)
[2025-06-20 17:48:18 -0500] [1275174] [INFO] Booting worker with pid: 1275174
[2025-06-20 17:48:18 -0500] [1270379] [INFO] Worker exiting (pid: 1270379)
[2025-06-20 17:48:18 -0500] [1270377] [INFO] Worker exiting (pid: 1270377)
[2025-06-20 17:48:19 -0500] [679494] [ERROR] Worker (pid:1270379) was sent SIGTERM!
[2025-06-20 17:48:19 -0500] [679494] [ERROR] Worker (pid:1270378) was sent SIGTERM!
[2025-06-20 17:48:19 -0500] [679494] [ERROR] Worker (pid:1270377) was sent SIGTERM!
[2025-06-20 17:53:52 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 17:53:52 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 17:53:52 -0500] [1275244] [INFO] Booting worker with pid: 1275244
[2025-06-20 17:53:52 -0500] [1275245] [INFO] Booting worker with pid: 1275245
[2025-06-20 17:53:52 -0500] [1275172] [INFO] Worker exiting (pid: 1275172)
[2025-06-20 17:53:52 -0500] [1275174] [INFO] Worker exiting (pid: 1275174)
[2025-06-20 17:53:52 -0500] [1275173] [INFO] Worker exiting (pid: 1275173)
[2025-06-20 17:53:52 -0500] [1275246] [INFO] Booting worker with pid: 1275246
[2025-06-20 17:53:52 -0500] [679494] [ERROR] Worker (pid:1275174) was sent SIGTERM!
[2025-06-20 17:53:52 -0500] [679494] [ERROR] Worker (pid:1275172) was sent SIGTERM!
[2025-06-20 17:53:52 -0500] [679494] [ERROR] Worker (pid:1275173) was sent SIGTERM!
[2025-06-20 17:55:59 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 17:55:59 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 17:55:59 -0500] [1275623] [INFO] Booting worker with pid: 1275623
[2025-06-20 17:55:59 -0500] [1275624] [INFO] Booting worker with pid: 1275624
[2025-06-20 17:55:59 -0500] [1275244] [INFO] Worker exiting (pid: 1275244)
[2025-06-20 17:55:59 -0500] [1275245] [INFO] Worker exiting (pid: 1275245)
[2025-06-20 17:55:59 -0500] [1275246] [INFO] Worker exiting (pid: 1275246)
[2025-06-20 17:55:59 -0500] [1275625] [INFO] Booting worker with pid: 1275625
[2025-06-20 17:56:00 -0500] [679494] [ERROR] Worker (pid:1275246) was sent SIGTERM!
[2025-06-20 17:59:40 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 17:59:40 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 17:59:40 -0500] [1275641] [INFO] Booting worker with pid: 1275641
[2025-06-20 17:59:40 -0500] [1275642] [INFO] Booting worker with pid: 1275642
[2025-06-20 17:59:40 -0500] [1275624] [INFO] Worker exiting (pid: 1275624)
[2025-06-20 17:59:40 -0500] [1275623] [INFO] Worker exiting (pid: 1275623)
[2025-06-20 17:59:40 -0500] [1275625] [INFO] Worker exiting (pid: 1275625)
[2025-06-20 17:59:40 -0500] [1275643] [INFO] Booting worker with pid: 1275643
[2025-06-20 17:59:41 -0500] [679494] [ERROR] Worker (pid:1275625) was sent SIGTERM!
[2025-06-20 17:59:41 -0500] [679494] [ERROR] Worker (pid:1275624) was sent SIGTERM!
[2025-06-20 17:59:41 -0500] [679494] [ERROR] Worker (pid:1275623) was sent SIGTERM!
[2025-06-20 18:01:46 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 18:01:46 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 18:01:46 -0500] [1275659] [INFO] Booting worker with pid: 1275659
[2025-06-20 18:01:46 -0500] [1275660] [INFO] Booting worker with pid: 1275660
[2025-06-20 18:01:46 -0500] [1275641] [INFO] Worker exiting (pid: 1275641)
[2025-06-20 18:01:46 -0500] [1275642] [INFO] Worker exiting (pid: 1275642)
[2025-06-20 18:01:46 -0500] [1275643] [INFO] Worker exiting (pid: 1275643)
[2025-06-20 18:01:46 -0500] [1275661] [INFO] Booting worker with pid: 1275661
[2025-06-20 18:06:40 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 18:06:40 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 18:06:40 -0500] [1276054] [INFO] Booting worker with pid: 1276054
[2025-06-20 18:06:40 -0500] [1276055] [INFO] Booting worker with pid: 1276055
[2025-06-20 18:06:40 -0500] [1275659] [INFO] Worker exiting (pid: 1275659)
[2025-06-20 18:06:40 -0500] [1275660] [INFO] Worker exiting (pid: 1275660)
[2025-06-20 18:06:40 -0500] [1276056] [INFO] Booting worker with pid: 1276056
[2025-06-20 18:06:40 -0500] [1275661] [INFO] Worker exiting (pid: 1275661)
[2025-06-20 18:06:41 -0500] [679494] [ERROR] Worker (pid:1275661) was sent SIGTERM!
[2025-06-20 18:06:41 -0500] [679494] [ERROR] Worker (pid:1275660) was sent SIGTERM!
[2025-06-20 18:06:41 -0500] [679494] [ERROR] Worker (pid:1275659) was sent SIGTERM!
[2025-06-20 18:12:03 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 18:12:03 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 18:12:03 -0500] [1276090] [INFO] Booting worker with pid: 1276090
[2025-06-20 18:12:03 -0500] [1276091] [INFO] Booting worker with pid: 1276091
[2025-06-20 18:12:03 -0500] [1276056] [INFO] Worker exiting (pid: 1276056)
[2025-06-20 18:12:03 -0500] [1276054] [INFO] Worker exiting (pid: 1276054)
[2025-06-20 18:12:03 -0500] [1276055] [INFO] Worker exiting (pid: 1276055)
[2025-06-20 18:12:03 -0500] [1276092] [INFO] Booting worker with pid: 1276092
[2025-06-20 18:12:04 -0500] [679494] [ERROR] Worker (pid:1276055) was sent SIGTERM!
[2025-06-20 18:12:04 -0500] [679494] [ERROR] Worker (pid:1276056) was sent SIGTERM!
[2025-06-20 18:12:04 -0500] [679494] [ERROR] Worker (pid:1276054) was sent SIGTERM!
[2025-06-20 18:19:12 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 18:19:12 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 18:19:12 -0500] [1276507] [INFO] Booting worker with pid: 1276507
[2025-06-20 18:19:12 -0500] [1276508] [INFO] Booting worker with pid: 1276508
[2025-06-20 18:19:12 -0500] [1276090] [INFO] Worker exiting (pid: 1276090)
[2025-06-20 18:19:12 -0500] [1276091] [INFO] Worker exiting (pid: 1276091)
[2025-06-20 18:19:12 -0500] [1276092] [INFO] Worker exiting (pid: 1276092)
[2025-06-20 18:19:12 -0500] [1276509] [INFO] Booting worker with pid: 1276509
[2025-06-20 18:19:13 -0500] [679494] [ERROR] Worker (pid:1276090) was sent SIGTERM!
[2025-06-20 18:19:13 -0500] [679494] [ERROR] Worker (pid:1276092) was sent SIGTERM!
[2025-06-20 18:19:13 -0500] [679494] [ERROR] Worker (pid:1276091) was sent SIGTERM!
[2025-06-20 18:27:09 -0500] [679494] [INFO] Handling signal: hup
[2025-06-20 18:27:09 -0500] [679494] [INFO] Hang up: Master
[2025-06-20 18:27:09 -0500] [1276917] [INFO] Booting worker with pid: 1276917
[2025-06-20 18:27:09 -0500] [1276918] [INFO] Booting worker with pid: 1276918
[2025-06-20 18:27:09 -0500] [1276508] [INFO] Worker exiting (pid: 1276508)
[2025-06-20 18:27:09 -0500] [1276507] [INFO] Worker exiting (pid: 1276507)
[2025-06-20 18:27:09 -0500] [1276509] [INFO] Worker exiting (pid: 1276509)
[2025-06-20 18:27:09 -0500] [1276919] [INFO] Booting worker with pid: 1276919
[2025-06-20 18:27:10 -0500] [679494] [ERROR] Worker (pid:1276508) was sent SIGTERM!
[2025-06-20 18:27:10 -0500] [679494] [ERROR] Worker (pid:1276509) was sent SIGTERM!
[2025-06-20 18:27:10 -0500] [679494] [ERROR] Worker (pid:1276507) was sent SIGTERM!
[2025-06-20 18:50:38 -0500] [1276917] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-20 22:53:41 -0500] [1276917] [WARNING] Invalid request from ip=206.168.34.72: Invalid HTTP Version: (2, 0)
[2025-06-21 02:15:03 -0500] [1276917] [WARNING] Invalid request from ip=167.94.146.57: Invalid HTTP Version: (2, 0)
[2025-06-21 02:15:07 -0500] [1276917] [WARNING] Invalid request from ip=167.94.146.57: Invalid HTTP Version: (2, 0)
[2025-06-21 03:08:23 -0500] [1276919] [WARNING] Invalid request from ip=3.134.148.59: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-21 04:25:24 -0500] [1276918] [WARNING] Invalid request from ip=185.156.73.154: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 05:42:39 -0500] [1276918] [WARNING] Invalid request from ip=185.156.73.154: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 07:20:09 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1276919)
[2025-06-21 07:20:09 -0500] [1276919] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-21 07:20:09 -0500] [1276919] [INFO] Worker exiting (pid: 1276919)
[2025-06-21 07:20:09 -0500] [1310277] [INFO] Booting worker with pid: 1310277
[2025-06-21 07:26:05 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1276917)
[2025-06-21 07:26:05 -0500] [1276917] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 271, in parse
    self.get_data(unreader, buf, stop=True)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-21 07:26:05 -0500] [1276917] [INFO] Worker exiting (pid: 1276917)
[2025-06-21 07:26:05 -0500] [1310818] [INFO] Booting worker with pid: 1310818
[2025-06-21 09:25:16 -0500] [1276918] [WARNING] Invalid request from ip=185.243.96.117: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 09:31:02 -0500] [1310818] [WARNING] Invalid request from ip=185.243.96.117: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 09:55:47 -0500] [1276918] [WARNING] Invalid request from ip=185.243.96.117: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 10:02:37 -0500] [1276918] [WARNING] Invalid request from ip=185.243.96.117: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 10:27:20 -0500] [1310818] [WARNING] Invalid request from ip=185.243.96.117: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 10:31:49 -0500] [1310277] [WARNING] Invalid request from ip=185.243.96.117: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 12:44:11 -0500] [1310818] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 15:37:15 -0500] [1276918] [WARNING] Invalid request from ip=185.170.144.3: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 18:48:41 -0500] [1276918] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-21 19:36:30 -0500] [1310277] [WARNING] Invalid request from ip=3.130.96.91: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-22 02:57:29 -0500] [1310818] [WARNING] Invalid request from ip=206.168.34.198: Invalid HTTP Version: (2, 0)
[2025-06-22 02:57:43 -0500] [1310277] [WARNING] Invalid request from ip=206.168.34.198: Invalid HTTP Version: (2, 0)
[2025-06-22 02:58:28 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1276918)
[2025-06-22 02:58:28 -0500] [1276918] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-22 02:58:28 -0500] [1276918] [INFO] Worker exiting (pid: 1276918)
[2025-06-22 02:58:29 -0500] [1359285] [INFO] Booting worker with pid: 1359285
[2025-06-22 05:48:41 -0500] [1310277] [WARNING] Invalid request from ip=167.94.138.193: Invalid HTTP Version: (2, 0)
[2025-06-22 05:48:51 -0500] [1310277] [WARNING] Invalid request from ip=167.94.138.193: Invalid HTTP Version: (2, 0)
[2025-06-22 05:49:30 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1310277)
[2025-06-22 05:49:30 -0500] [1310277] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-22 05:49:30 -0500] [1310277] [INFO] Worker exiting (pid: 1310277)
[2025-06-22 05:49:31 -0500] [1366121] [INFO] Booting worker with pid: 1366121
[2025-06-22 11:43:21 -0500] [1359285] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-22 13:50:51 -0500] [1359285] [WARNING] Invalid request from ip=162.142.125.120: Invalid HTTP Version: (2, 0)
[2025-06-22 13:50:54 -0500] [1366121] [WARNING] Invalid request from ip=162.142.125.120: Invalid HTTP Version: (2, 0)
[2025-06-22 13:51:28 -0500] [679494] [CRITICAL] WORKER TIMEOUT (pid:1366121)
[2025-06-22 13:51:28 -0500] [1366121] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
          ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
           ^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
        ^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/benjamin_gutierrez_3011/IOT_Riego/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
SystemExit: 1
[2025-06-22 13:51:28 -0500] [1366121] [INFO] Worker exiting (pid: 1366121)
[2025-06-22 13:51:28 -0500] [1385944] [INFO] Booting worker with pid: 1385944
[2025-06-22 17:14:02 -0500] [1359285] [WARNING] Invalid request from ip=199.45.155.71: Invalid HTTP Version: (2, 0)
[2025-06-22 17:14:07 -0500] [1310818] [WARNING] Invalid request from ip=199.45.155.71: Invalid HTTP Version: (2, 0)
[2025-06-22 17:42:20 -0500] [1385944] [WARNING] Invalid request from ip=147.45.112.101: Invalid HTTP request line: '\x03\x00\x00/*Ã \x00\x00\x00\x00\x00Cookie: mstshash=Administr'
[2025-06-23 00:33:46 -0500] [1359285] [WARNING] Invalid request from ip=3.131.215.38: Invalid HTTP request line: 'SSH-2.0-Go'
[2025-06-23 08:58:46 -0500] [1310818] [WARNING] Invalid request from ip=167.94.146.60: Invalid HTTP Version: (2, 0)
[2025-06-23 08:58:58 -0500] [1310818] [WARNING] Invalid request from ip=167.94.146.60: Invalid HTTP Version: (2, 0)
